IMDA-BTG/aiverify-developer-tools
# Implemented integration for ai-verify-plugin and cookiecutter

See https://github.com/IMDA-BTG/aiverify-developer-tools/issues/12.

To test:
Install tool as usual.
1. Use 'ai-verify-plugin gp' command to generate new plugin
2. Use 'ai-verify-plugin ga' command to generate new algo component under the new plugin.
3. Update algo codes and use 'ai-verify-plugin testa' command to test the new algo'
4. Use 'ai-verify-plugin zip' command to create the plugin zip.
Make sure the zip includes the algorithm.
